sql/changes/1.6/country_date_format.sql

```diff
@@ -0,0 +1,267 @@
+ALTER TABLE country
+  ADD date_format TEXT;

```

Comment:
Hi, wondering why we need this. Dojo knows all this, correct? If so, can't we leverage that?

Reply:
I still don't like this file. It'll be subject to aging and bitrot. And what's more: we can't change it after the fact, because that changes the sha sum.